[Outlook] (manifest) update the unified manifest quickstart (#4130)
OfficeDev/office-js-docs-pr: docs/quickstarts/outlook-quickstart-json-manifest.md

    </main>
    ```

### Try it out

[!INCLUDE [alert use https](../includes/alert-use-https.md)]

1. Open a command prompt *as an administrator* and run the following command in the root directory of your project. When you run this command, the local web server starts and your add-in will be [sideloaded](../outlook/sideload-outlook-add-ins-for-testing.md).

    ```command&nbsp;line
    npm start
    ```

    > [!NOTE]

>     > If this is the first time you've created an add-in on the computer, or the first time in over a month, you will be prompted to install security certificates.

Reviewer: ```suggestion
    > If this is the first time you've created an add-in on the computer, or the first time in over a month, you'll be prompted to install security certificates.
```
